src/thor/multimodal.cc

```diff
@@ -92,6 +92,8 @@ void MultiModalPathAlgorithm::Clear() {
 
   // Set the ferry flag to false
   has_ferry_ = false;
+  // Set not thru pruning to true
+  not_thru_pruning_ = true;
```

Comment:
aren't multi-modal routes always forward search (not bidirectional) - and we don't use not_thru pruning except in bidirectional A*?